examples/csv-string-to-record-array/csv_string_to_record_array.bal

```diff
@@ -0,0 +1,29 @@
+import ballerina/data.csv;
+import ballerina/io;
+
+type Book record {
+    string name;
+    string author;
+    decimal price;
+    string publishedDate;
+};
+
+type BriefBookInfo record {|
+    string name;
+    string author;
+|};
+
+public function main() {
+    string csvString = string `name,author,price,publishedDate
+                               Effective Java,Joshua Bloch,45.99,2018-01-01
+                               Clean Code,Robert C. Martin,37.50,2008-08-01`;
+
+    // Convert the CSV string to a record array
```

Comment:
Why are we still using convert instead of parse? 

- record array -> array of records
- add fullstops at the end